examples/ota-provider-app/linux/main.cpp

```diff
@@ -165,12 +170,7 @@ bool HandleOptions(const char * aProgram, OptionSet * aOptions, int aIdentifier,
         }
         break;
     case kOptionQueryImageStatus:
-        if (aValue == NULL)
```

Comment:
These checks are not necessary here since `ArgParser` will check that required arguments are supplied